content/en/apps/guides/data/csv-to-docs.md

```diff
@@ -295,7 +295,7 @@ To fetch the files from Google Drive run the `fetch-csvs-from-google-drive` acti
 
 #### Linking to Google Drive
 
-The file `csvs-on-google-drive.json` in your project's home directory will consist of a key value pair for each CSV file. The keys must be the filename where the CSV will be stored locally — see the CSV [file documentation](https://github.com/medic/medic-conf#csv-file-name) for the notation. The value of each key must be the ID of the corresponding file in Google Drive — the ID can be obtained from the URL eg `https://docs.google.com/spreadsheets/d/{FILE_ID}/edit`. 
+The file `csvs-on-google-drive.json` in your project's home directory will consist of a key value pair for each CSV file. The keys must be the filename where the CSV will be stored locally — see the CSV [file documentation](https://github.com/medic/medic-conf#user-content-csv-file-name) for the notation. The value of each key must be the ID of the corresponding file in Google Drive — the ID can be obtained from the URL eg `https://docs.google.com/spreadsheets/d/{FILE_ID}/edit`. 
```

Comment:
this looks right but just takes me to the top of the medic-conf page
https://github.com/medic/medic-conf#csv-file-name

Reply:
oh - so it does! I'm not actually sure if there's a logical link on this page for "file documentation". I'd guess ["managing data and images"](https://github.com/medic/medic-conf#managing-data-and-images) maybe?

Reply:
or maybe "[csv to docs](https://docs.communityhealthtoolkit.org/apps/guides/data/csv-to-docs/)" ?
@abbyad do you know?

Reply:
@MaxDiz, that's the file in question, right? I think that's a good start, but I would go with the [#creating-csv-files-for-contacts-reports](https://docs.communityhealthtoolkit.org/apps/guides/data/csv-to-docs/#creating-csv-files-for-contacts-reports) anchor. In general, if we are linking to documentation content in our own github repos it probably means that we need to port content over to the doc site. I think in this case we already did!

Reply:
ack!  So sorry I missed this response.  I committed the suggested link -  thank you!